client/rest-high-level/src/test/java/org/elasticsearch/client/CrudIT.java

```diff
@@ -265,4 +340,27 @@ public void testIndex() throws IOException {
                          "version conflict, document already exists (current version [1])]", exception.getMessage());
         }
     }
+
+    /**
+     * @return A sample index request
+     * @throws IOException If problem when generating the content
+     */
+    private IndexRequest createSampleIndexRequest() throws IOException {
```

Comment:
> don't we always do createSampleDocument(createSampleIndexRequest))?

No. We also do:

```java
createSampleDocument(createSampleIndexRequest().routing("foo"));
```

See https://github.com/elastic/elasticsearch/blob/5aa2ab3bbc2a0bc2dca1e1399e6dada5907d24c2/client/rest-high-level/src/test/java/org/elasticsearch/client/CrudIT.java#L112-L112